libs/labelbox/tests/data/annotation_types/test_annotation.py

```diff
@@ -1,7 +1,8 @@
 import pytest
 
-from labelbox.data.annotation_types import (Text, Point, Line,
+from labelbox.data.annotation_types import (Text, Point, Line, PromptText,
```

Comment:
is there an actual test for this?

Reply:
No this was a typo on this file I original had my serialization tests here but put it on a separate file